scripts/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Is it worth changing the file permissions on this so it is executable e.g. `chmod +x scripts/deploy.sh`